component-library/src/app/pages/date-picker-documentation/date-picker-documentation.component.scss

```diff
@@ -5,26 +5,24 @@
 
 @include page.template;
 
-.usage-figma {
-  .access-input-1-2 {
-    @include grid.up-laptop-layout {
-      padding-left: 22px;
+.block {
+  &__usage-figma {
+    .access-input-1-2 {
+      @include grid.up-laptop-layout {
+        padding-left: 22px;
+      }
```

Comment:
with the standardization of templates can this be done from the template css file?

Reply:
Since they are too specific I kept them here - but we can move it for sure!